### PR TITLE
Fixing typo in Writing_Native_Bears.rst

### PR DESCRIPTION
**Pull request by shivank1234**
Writing_Native_Bears.rst:Fix typo on line 205
This fixes typo and removes the extra 
us on line 205 in coala documentation.
Fixes https://github.com/coala/coala/issues/5766

### CODE REVIEW

File: docs/Developers/Writing_Native_Bears.rst [hunk 1]
```diff
@@ -202,7 +202,7 @@ First, coala looked at the parameters of the run method and found that
 we need some value named user\_input. Then it parsed our documentation
 comment and found a description for the parameter which was shown to us
 to help us choose the right value. After the needed values are provided,
-coala converts us the value into a string because we've provided the
+coala converts  the value into a string because we've provided the
```
Comment (Man-Jain) [inline docs/Developers/Writing_Native_Bears.rst]: 1. The space is double here. Remove one space.

**Comment by Sidsharik** [comment]
Please change your commit message to adhere with the coala commit guidelines. Refer this - (coala.io/commit). And there is an extra space in your edit, remove that and you're good to go  :)

File: docs/Developers/Writing_Native_Bears.rst (hunk 1 above)
Comment (shivank1234) [reply docs/Developers/Writing_Native_Bears.rst]: Thanks for the reply.

**Comment by akshatkarani** [comment]
As someone pointed above squash your commits to one. Also use closes instead of fixes as this is not a bug.

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**